lineman/app/components/group_page/cover_photo_form/cover_photo_form.haml

```diff
@@ -0,0 +1,21 @@
+.group-cover-photo-modal
+  .lmo-disabled-form{ng-show: 'isDisabled'}
+  .modal-header
+    %button{type: 'button', class: 'close', ng-click: '$close()'}
+      %span{aria-hidden: true} &times;
```

Comment:
oh I did a search and replace on all the source earlier in the week. Yes indeed on _something_ that allows reuse of the same template.